code/modules/projectiles/projectile.dm

```diff
@@ -185,7 +186,7 @@
 				playsound(loc, hitsound, volume, 1, -1)
 			L.visible_message("<span class='danger'>[L] is hit by \a [src][organ_hit_text]!</span>", \
 					"<span class='userdanger'>[L] is hit by \a [src][organ_hit_text]!</span>", null, COMBAT_MESSAGE_RANGE)
-		if(def_zone == BODY_ZONE_HEAD)
+		if(candink && def_zone == BODY_ZONE_HEAD)

```

Comment:
This could as well just be
```
		if(!suppressed && !nodamage && flag == "bullet" && def_zone == BODY_ZONE_HEAD)
```
Although having a var for it isn't necessarily a bad thing.